public/403.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <title>ERROR 403</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="mainC">
+  <div class="message">
+    <h1>403</h1>
+    <h3>Forbidden | The Browser you're currently using does not supports <strong>susi.ai</strong></h3>
```

Comment:
There's a slight grammatical error here. Just change "supports" to "support"

Reply:
Oh. :sweat_smile: 

Done :+1: